media/img/firefox/mobile/arrow-down.svg

```diff
@@ -0,0 +1,19 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg width="26px" height="28px" viewBox="0 0 26 28" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
+    <!-- Generator: Sketch 47.1 (45422) - http://www.bohemiancoding.com/sketch -->
```

Comment:
Worth throwing these SVG's through [SVGOMG](https://jakearchibald.github.io/svgomg/)

Reply:
D'oh! Thought I did that already. Done!